test/object-store/util/sync/redirect_server.hpp

```diff
@@ -97,11 +171,11 @@ class RedirectingHttpServer {
 
     struct Conn : public util::RefCountBase, websocket::Config {
         Conn(network::Service& service, const std::shared_ptr<util::Logger>& logger)
-            : random(Catch::getSeed())
-            , logger(logger)
+            : logger(logger)
             , socket(service)
             , http_server(socket, logger)
         {
+            util::seed_prng_nondeterministically(random); // Throws
```

Comment:
Why do we need to seed this non-deterministically? If you just use the catch seed then you can actually replay the random values when you get an error?